realm/realm-library/src/objectServer/java/io/realm/mongodb/AppConfiguration.java

```diff
@@ -294,7 +295,7 @@ public Builder(String appId) {
          * @param key a 64 byte encryption key.
          * @throws IllegalArgumentException if the key is not 64 bytes long.
          */
-        public Builder encryptionKey(byte[] key) {
+        public Builder encryptionKey(@ParametersAreNonnullByDefault byte[] key) {
```

Comment:
We should create an issue about moving this to a package level annotation instead of only here. Generally all our public API should be non-null by default to play nicer with Kotlin.

This is fine for this PR though.